input/kafka.go

```diff
@@ -151,6 +184,9 @@ func (processor *KafkaProcessor) Cleanup(sarama.ConsumerGroupSession) error {
 // ConsumeClaim must start a consumer loop of ConsumerGroupClaim's Messages().
 func (processor *KafkaProcessor) ConsumeClaim(session sarama.ConsumerGroupSession, claim sarama.ConsumerGroupClaim) error {
 	for message := range claim.Messages() {
+		if message.Offset%1000 == 0 {
```

Comment:
You also apply a modulus within the function, I guess you may want to keep only one

Reply:
Done.